CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # MicroProfile Language Server Changelog
 
+## [0.12.0](https://github.com/eclipse/lsp4mp/milestone/13?closed=1) (August 14, 2024)
+
+### Enhancements
+
+- Add support for @Startup healthcheck diagnostic. See [#443](https://github.com/eclipse/lsp4mp/issues/443).
```

Comment:
add #456

Reply:
fixed by adding #448 (the issue and not the PR)